database/ffldb/ff/rollback.go

```diff
@@ -113,11 +118,24 @@ func (s *flatFileStore) rollback(targetLocation *flatFileLocation) error {
 	return nil
 }
 
-// deleteFile removes the file for the passed flat file number. The file must
-// already be closed and it is the responsibility of the caller to do any
-// other state cleanup necessary.
+// deleteFile removes the file for the passed flat file number.
```

Comment:
Say in the comment that the lruMutex and openFilesMutex have to be held by the caller